main.py

```diff
@@ -9,49 +9,37 @@
 from influx import InfluxConnector
 from life360 import Life360Connector
 from pathlib import Path
+from config import Config
```

Comment:
Is it a new file?  Should it be added to the PR?

Reply:
oops. Added